sdk/ai/azure-ai-projects/samples/agents/sample_agents_enterprise_file_search.py

```diff
@@ -0,0 +1,78 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+FILE: sample_agents_vector_store_batch_file_search_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to add files to agent during the vector store creation.
+
+USAGE:
+    python sample_agents_vector_store_file_search.py
+
+    Before running the sample:
+
+    pip install azure.ai.projects azure-identity azure-ai-ml
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+
+import os
+from azure.ai.projects import AIProjectClient
+from azure.ai.projects.models import FileSearchTool, VectorStoreDataSource, VectorStoreDataSourceAssetType
+from azure.identity import DefaultAzureCredential
+
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
+credential = DefaultAzureCredential()
+project_client = AIProjectClient.from_connection_string(
+    credential=credential, conn_str=os.environ["PROJECT_CONNECTION_STRING"]
+)
+
+with project_client:
+
+    # We will upload the local file to Azure and will use it for vector store creation.
+    _, asset_uri = project_client.upload_file("./product_info_1.md")
+
+    # create a vector store with no file and wait for it to be processed
+    ds = VectorStoreDataSource(storage_uri=asset_uri, asset_type=VectorStoreDataSourceAssetType.URI_ASSET)
```

Comment:
Is storage_uri correct? VectorStoreDataSource has asset_identifier

Reply:
Replaced; I have forgot to change it in samples.